modules/clients/main.tf

```diff
@@ -113,7 +113,13 @@ resource "aws_launch_template" "this" {
       })
     }
   }
-  user_data  = local.custom_data
+  user_data = local.custom_data
+  lifecycle {
+    precondition {
+      condition     = var.alb_dns_name == null ? var.weka_cluster_size > 0 : true
+      error_message = "Please povide weka_cluster size in case of not using ALB"
```

Comment:
this is weird requirement, we probably can get info we need here in a different way...i understand this PR does not add it, but just validates

Reply:
why it needs to know the size? (did not go to see how variable is used..)